.github/workflows/managed-runtime-base-qualification.yaml

```
@@ -0,0 +1,611 @@
+      contents: read
+      packages: read
+    env:
+      BASE_SHA: ${{ needs.authenticate-source.outputs.base_sha }}
+      CANDIDATE_SHA: ${{ needs.authenticate-source.outputs.candidate_sha }}
+      E2E_ARTIFACT_DIR: ${{ github.workspace }}/trusted/e2e-artifacts/live/managed-image-activation
+      E2E_JOB: "1"
+      E2E_TARGET_ID: managed-image-activation
+      NEMOCLAW_CLI_BIN: ${{ github.workspace }}/candidate/bin/nemoclaw.js
+      NEMOCLAW_E2E_EXPECTED_SHA: ${{ needs.authenticate-source.outputs.candidate_sha }}
+      NEMOCLAW_E2E_SHARD: default
+      NEMOCLAW_MANAGED_ACTIVATION_CATALOG: ${{ runner.temp }}/managed-runtime-candidate/catalog.json
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)